src/main/java/net/minecraftforge/common/config/ConfigManager.java

```diff
@@ -218,39 +218,47 @@ private static void sync(Configuration cfg, Class<?> cls, String modid, String c
     {
         for (Field f : cls.getDeclaredFields())
         {
-            if (!Modifier.isPublic(f.getModifiers()))
+            if (!Modifier.isPublic(f.getModifiers())) //Allow modders to have private field for internal data
                 continue;
```

Comment:
Remove all your useless comment spam.

Reply:
I'll gladly do, but when I tried to understand this code (which I wrote almost a year ago), I could barely understand it anymore. As I suspect that there will be more bugs and/or missing features, other people might want to contribute. I think it is better to comment the code so that strangers can understand.

However, if you insist, I will remove it all.

Reply:
The only confusion is that you didnt clearly specify what the key should be.
But things like 'getType() Returns the type!' is just redundant.
Your code should be self-documenting for the most part sprinkeled with a few "I couldn't specify the format of the key in the variable name so here is the correct format" style comments.

Reply:
OK, I will try to change the comments to fit your criteria.